marl_eval/json_tools/pull_neptune_data.py

```diff
@@ -0,0 +1,54 @@
+# python3
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from typing import List
+
+import neptune
+from colorama import Fore, Style
+from tqdm import tqdm
+
+
+def pull_neptune_data(
+    project_name: str, system_name: str, tag: List, store_directory: str = "."
+) -> None:
+    """Pulls the experiments data from Neptune to local directory.
+
+    Args:
+        project_name (str): Name of the Neptune project.
+        system_name (str): Name of the system (example: ff-ippo).
+        tag (List): List of tags.
+        store_directory (str, optional): Directory to store the data.
+    """
+    # Get the run ids
+    project = neptune.init_project(project=project_name)
+    runs_table_df = project.fetch_runs_table(state="inactive", tag=tag).to_pandas()
+    runs_table_df = runs_table_df[
+        runs_table_df["config/logger/system_name"] == system_name
```

Comment:
Similar here, we don't have to filter by system name here. It hopefully will also future proof things a bit where we are not tied to the way that the system names are added to the Neptune logs.